search-sentiments-map/public/index.html

```diff
@@ -9,62 +9,68 @@
-                <div role="tabpanel" class="tab-pane fade" id="top-trends-tab">Insert top trends here.</div>
+    <div class="content">
+      <!-- Bootstrap CSS. -->
+      <div class="modal fade" id="region-info-modal" tabindex="-1" role="dialog" aria-hidden="true">
+        <div class="modal-dialog">
+          <div class="modal-content">
+            <div class="modal-header">
+              <h4 class="modal-title" id="modal-title"></h4>
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+            </div>
+            <div class="modal-body">
+              <div role="tabpanel">
+                <ul class="nav nav-tabs" role="tablist">
+                  <li role="presentation" class="nav-item active">
+                    <a class="nav-link active" href="#search-results-tab" aria-controls="search-results-tab" role="tab" data-toggle="tab">Search Results</a>
+                  </li>
+                  <li role="presentation" class="nav-item">
+                    <a class="nav-link" href="#top-trends-tab" aria-controls="top-trends-tab" role="tab" data-toggle="tab">Top Trends</a>
+                  </li>
+                </ul>
+                <div class="tab-content">
+                  <div role="tabpanel" class="tab-pane fade show active" id="search-results-tab"></div>
+                  <div role="tabpanel" class="tab-pane fade" id="top-trends-tab"></div>
+                </div>
               </div>
             </div>
           </div>
         </div>
       </div>
-    </div>
-    <h2 id="title">Sentiment Map for Google Search</h2>
-    <div class="row">
-      <div class="col-sm-9">
-        <div id="map"></div>
-        <div id="legend">
-          <div id="data-min">min</div>
-          <div class="color-key"></div>
-          <div id="data-max">max</div>
+
+      <h2 id="title">Sentiment Map for Google Search</h2>
+      <div class="row">
+        <div class="col-sm-9">
+          <h5 id="topic-header">Worldwide sentiments</h5>
+          <div id="map"></div>
+          <div id="legend">
+            <div id="data-min">min</div>
+            <div class="color-key"></div>
+            <div id="data-max">max</div>
+          </div>
         </div>
-      </div>
-      <div class="col-sm-3">
-        <form id="search-box" class="input-group row" action="">
-          <input type="text" class="form-control col-sm-9" placeholder="Search a Topic" name="search">
-          <div class="input-group-append">
-            <button type="submit"><i class="fa fa-search"></i></button>
+        <div class="col-sm-3">
+          <form id="search-box" class="input-group row" action="">
+            <input type="text" class="form-control col-sm-9" placeholder="Search a Topic" name="search">
+            <div class="input-group-append">
+              <button type="submit"><i class="fa fa-search"></i></button>
+            </div>
+          </form>
```

Comment:
Carmen and I are using newsentiment version search box can you replace with this
        <div id="search-box" class="input-group row">
          <textarea class="form-control col-sm-9" id="search-topic" name="search-topic" placeholder="Search for a topic..." rows="1"></textarea>
          <br/>
          <div class="input-group-append">
            <button onclick="getSentiment()" type="submit"><i class="fa fa-search"></i></button>
          </div>  
        </div>
        <div id="sentiment-score"></div>